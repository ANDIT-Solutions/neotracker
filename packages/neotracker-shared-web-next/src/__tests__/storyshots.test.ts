// tslint:disable no-import-side-effect ordered-imports
import 'jest-styled-components';
import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  configPath: '.storybook/config.snap.js',
  storyNameRegex: /^.*\[snapshot\].*$/,
});
