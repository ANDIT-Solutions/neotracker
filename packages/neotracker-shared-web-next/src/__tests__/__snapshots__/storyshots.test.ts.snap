// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots components/singel/Box Hello Box [snapshot] 1`] = `
.c2 {
  margin: 0;
  padding: 0;
  border: 0;
  font-size: 100%;
  font-family: inherit;
  vertical-align: baseline;
  box-sizing: border-box;
}

.c1 {
  margin: 0;
  padding: 0;
  border: 0;
  font-size: 100%;
  font-family: inherit;
  vertical-align: baseline;
  box-sizing: border-box;
}

.c0 {
  border: 1px solid rgba(0,0,0,0.3);
  border-radius: 0.25em;
}

<div
  className="c0 c1 c2"
>
  Hello Box
</div>
`;
